src/main/java/com/automationpractice/BasePage.java

```diff
@@ -0,0 +1,34 @@
+package com.automationpractice;
+
+import org.apache.log4j.Logger;
+import org.openqa.selenium.JavascriptExecutor;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+public class BasePage {
+
+    Logger log = Logger.getLogger("applicationLogger");
+
+    public void jsShowSuccessfulLoginPopup(WebDriver driver) {
+        JavascriptExecutor js = (JavascriptExecutor) driver;
+        js.executeScript("alert('Message generated by JavscriptExecutor: Log in operation is successful!')");
+    }
+
+    public void jsScrollToBottom(WebDriver driver) {
+        log.debug("Scrolling page down");
+        JavascriptExecutor js = (JavascriptExecutor) driver;
+        js.executeScript("window.scrollTo(0, document.body.scrollHeight)");
+    }
+
+    public void jsScrollToTop(WebDriver driver) {
+        log.debug("Scrolling page up");
+        JavascriptExecutor js = (JavascriptExecutor) driver;
+        js.executeScript("window.scrollTo(document.body.scrollHeight, 0)");
+    }
+
+    public void jsHighlightElement(WebDriver driver, WebElement element) {
+        log.debug("Highlighting element");
+        JavascriptExecutor js = (JavascriptExecutor) driver;
+        js.executeScript("arguments[0].setAttribute('style', 'background: yellow; border: 2px solid red;');", element);
+    }
```

Comment:
It's utils methods that should be used **only** in pages so they should be private